sigstore-gradle/sigstore-gradle-sign-base-plugin/src/main/kotlin/dev/sigstore/sign/SigstoreSignExtension.kt

```diff
@@ -88,28 +91,34 @@ abstract class SigstoreSignExtension(private val project: Project) {
         val artifacts = mutableMapOf<PublicationArtifact, T>()
         publication.allPublishableArtifacts {
             val publishableArtifact = this
-            if (file.extension !in listOf("asc", SigstoreSignature.EXTENSION)) {
+            if (!file.name.endsWith(".asc") and !file.name.endsWith(SigstoreSignature.EXTENSION)) {
```

Comment:
```suggestion
            if (!file.name.endsWith(".asc") && !file.name.endsWith(SigstoreSignature.EXTENSION)) {
```

`bool and bool` doesn't short-curcuit, so regular `&&` is the way to go

Reply:
Sgtm